File: Project/MyProject/MyProject/Bak/Being/PlayerOtherMovement.cs
namespace SDK.Lib
{
    public class PlayerOtherMovement : BeingEntityMovement
    {
        public PlayerOtherMovement(SceneEntityBase entity)
            : base(entity)
        {

        }

        override public void onTick(float delta, TickMode tickMode)
        {
            base.onTick(delta, tickMode);
        }
    }
}

File: Project/MyProject/MyProject/Source/MyProject/Private/BasicFrame/Module/Common/Scene/Being/PlayerOtherMovement.cpp
#include "MyProject.h"
#include "PlayerOtherMovement.h"
#include "MClassFactory.h"

MY_BEGIN_NAMESPACE(MyNS)

M_IMPLEMENT_AND_REGISTER_CLASS(PlayerOtherMovement, BeingEntityMovement)

PlayerOtherMovement::PlayerOtherMovement(SceneEntityBase* entity)
{
	Super(entity);
}

void PlayerOtherMovement::onTick(float delta, TickMode tickMode)
{
	Super::onTick(delta, tickMode);
}

MY_END_NAMESPACE